### PR TITLE
LPS-162686 | master

### PR DESCRIPTION
**Pull request by brookedalton**
https://issues.liferay.com/browse/LPS-162686

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:test:search:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/Blueprints.macro [hunk 1]
```diff
@@ -52,6 +52,18 @@ definition {
 			portletName = "Blueprints Options");
 	}
 
+	macro changeLocale {
+		Click(locator1 = "Blueprints#BLUEPRINTS_LOCALIZATION");
+
+		Click(
+			key_locale = "${locale}",
+			locator1 = "Translation#DROPDOWN_MENU_ITEM");
```
Comment (joshchong) [inline portal-web/test/functional/com/liferay/portalweb/macros/Blueprints.macro:60]: Can you replace these steps with the ones that are in PageEditor.changeLocale macro? You can remove the `Blueprints#BLUEPRINTS_LOCALIZATION` path after

**Review by joshchong — CHANGES_REQUESTED** [review]

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:forward

**Pull request closed without merging**